script.js

```diff
@@ -1,51 +1,63 @@
 /* eslint no-use-before-define: "off" */
-// Retrieving the books from localStorage or create a new empty array
-let allBooks = JSON.parse(localStorage.getItem('allBooks')) || [];
+/* eslint class-methods-use-this: "off" */
```

Comment:
- [ ] **Optional:** Kindly ensure you solve the linters rather than switch them off in future projects.

Reply:
how can we solve this?